src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -1001,6 +1001,30 @@
      "CREATE INDEX idx_resource_events_timestamp ON resource_events(timestamp)"
      "ALTER TABLE resource_events ADD CONSTRAINT resource_events_report_id_fkey FOREIGN KEY (report_id) REFERENCES reports(id) ON DELETE CASCADE"))
 
+(defn drop-latest-reports
```

Comment:
Yeah, I'd merge this up with the last migration if possible. We are already dropping and recreating latest_report in the last migration, seems silly to just then go and remove it.